sirepo/template/cloudmc.py

```diff
@@ -408,6 +401,65 @@ def _dagmc_animation_python(filename):
 """
 
 
+def _energy_plot(run_dir, data):
+    import openmc
+
+    def _bin(val, mesh, idx):
+        # mesh units are in cm
+        return numpy.floor(
+            mesh.dimension[idx]
+            * abs(val * 1e2 - mesh.lower_left[idx])
+            / abs(mesh.upper_right[idx] - mesh.lower_left[idx])
+        ).astype(int)
+
+    plots = []
+    tally_name = data.models.energyAnimation.tally
+    t = openmc.StatePoint(run_dir.join(_statepoint_filename(data))).get_tally(
+        name=tally_name
+    )
+    try:
+        e_f = t.find_filter(openmc.EnergyFilter)
+    except ValueError:
+        return PKDict(error=f"No energy filter defined for tally {tally_name}")
+
+    tally = _get_tally(data.models.settings.tallies, tally_name)
+    mesh = _get_filter(tally, "meshFilter")
+    e = _get_filter(tally, "energyFilter")
+    r = data.models.energyAnimation
+    mean = numpy.reshape(
+        getattr(t, "mean")[:, :, t.get_score_index(r.score)].ravel(),
+        (*mesh.dimension, -1),
+    )
+    # std_dev = getattr(t, "std_dev")[:, :, t.get_score_index(r.score)].ravel()
```

Comment:
comment needed?

Reply:
I think it is there because eventually the plot will have error bars based on the std_dev values.